Sources/KSCrashRecording/include/KSCrashAppMemory.h

```diff
@@ -131,7 +135,9 @@ typedef NS_ENUM(NSUInteger, KSCrashAppMemoryState) {
  * `KSCrashAppMemoryStateToString` returns a `const char*`
  * because it needs to be async safe.
  */
-FOUNDATION_EXPORT const char *KSCrashAppMemoryStateToString(KSCrashAppMemoryState state);
-FOUNDATION_EXPORT KSCrashAppMemoryState KSCrashAppMemoryStateFromString(NSString *const state);
+FOUNDATION_EXPORT const char *KSCrashAppMemoryStateToString(KSCrashAppMemoryState state) 
+NS_SWIFT_NAME(string(from:));
+FOUNDATION_EXPORT KSCrashAppMemoryState KSCrashAppMemoryStateFromString(NSString *const string)
+NS_SWIFT_NAME(memoryState(from:));
```

Comment:
nit: (it's a bit difficult to get if `NS_SWIFT_NAME` is for the line above or below)
```suggestion
FOUNDATION_EXPORT const char *KSCrashAppMemoryStateToString(KSCrashAppMemoryState state) 
NS_SWIFT_NAME(string(from:));

FOUNDATION_EXPORT KSCrashAppMemoryState KSCrashAppMemoryStateFromString(NSString *const string)
NS_SWIFT_NAME(memoryState(from:));
```